src/qibolab/platform/platform.py

```diff
@@ -109,6 +109,9 @@ class Platform:
     topology: nx.Graph = field(default_factory=nx.Graph)
     """Graph representing the qubit connectivity in the quantum chip."""
 
+    wire_names: List[Union[int, str]] = None
+    """List of physical qubit (wire) names."""
```

Comment:
Since you only assign this in the `__post_init__`, you could replace this attribute with a property.

Reply:
Btw, in this context, you should use a different name, since the *wire* is a circuit-related concept (and the platform is not directly related to circuits).

Since it's used just twice, you may even consider stripping this completely, and just use `list(platform.qubits)`.

Reply:
I also think we don't need to touch the platform in this PR. If the idea is that qibo wire names match platform qubit names, then we should only need to communicate the wire names to the compiler.